src/sentry/web/forms/accounts.py

```diff
@@ -553,16 +553,19 @@ def __init__(self, project, user, *args, **kwargs):
         # for notifications to keep their settings
         emails = [e.email for e in user.get_verified_emails()]
         alert_email = UserOption.objects.get_value(user=self.user, project=None, key='alert_email', default=None)
+        if alert_email:
+            alert_email = alert_email.rstrip()
```

Comment:
I'm okay with this but why are we expecting whitespace to be in the alert mail. Does this happen in practice? (Bad form normalization?) If yes, we should strip on both sides.